### PR TITLE
fix/remove-ssnrewardshare

### PR DESCRIPTION
**Pull request by teye**
## Description
This pr is a fix for a scilla bug which causes custom ADT not to be interpreted when used as a parameter. The pr replaces the `SsnRewardShare (ByStr20 Uint128)` ADT with `Pair ByStr20 Uint128`. Specifically, it replaces the existing `List SsnRewardShare` to `List (Pair ByStr20 Uint128)` for `proxy.scilla` and `ssnlist.scilla`.

## Affected Transition
`AssignStakeReward`

## Notes
This pr is a series of improvements as part of [ZIP-19 (WIP)](https://github.com/Zilliqa/ZIP/blob/feat/zip19/zips/zip-19.md)

### CODE REVIEW

**Comment by bb111189** [comment]
@jjcnn This PR is to fix the disambiguation bug for the staking contract

File: contracts/ssnlist.scilla [hunk 1]
```diff
@@ -1317,9 +1303,14 @@ end
 (***************************************************)
 
 (* @dev: Assign stake reward to all ssn from ssnlist. Used by verifier only. *)
-(* @param ssnrewardlist: List of SsnRewardShare *)
+(* @param ssnrewardlist: List of SSN address and cycle rewards in Qa *)
+(*  CycleReward       : Uint128 *)
+(*                      Integer representation of reward assigned by the verifier to this SSN for this cycle. *)
+(*                      It's floor(NumberOfDSEpochsInCurrentCycle * 110,000 * VerificationPassed) *)
+(* https://github.com/Zilliqa/ZIP/blob/zip-11-author-fix/zips/zip-11.md#verifier *)
```
Comment (bb111189) [inline contracts/ssnlist.scilla:1310]: use the master branch
seems broken link to me

**Review by bb111189 — CHANGES_REQUESTED** [review]
changes looks fine to me except for the broken link in the comments

**Review by bb111189 — APPROVED** [review]

**Review by AmritKumar — APPROVED** [review]

**Review by renlulu — APPROVED** [review]
Looks good to me. Do notice verifier need some change corresponding to this. @teye @ansnunez

**Pull request merged**